core/src/partition/mesh_partition/01_mesh_partition_structured.tpp

```diff
@@ -47,7 +47,7 @@ MeshPartition<FunctionSpace::FunctionSpace<MeshType, BasisFunctionType>,
              << ", localSizesOnPartitions_: " << localSizesOnPartitions_
              << ", beginElementGlobal_: " << beginElementGlobal_;
 
-  LOG(DEBUG) << *this;
+  // LOG(DEBUG) << *this;
```

Comment:
I see that `meshPartition()` is used a lot in the codebase. If you want to keep the `meshPartition()` logging, you could implement something like a `meshPartition().data()` to extract the array data.

Reply:
I agree, I have added an issue about it.